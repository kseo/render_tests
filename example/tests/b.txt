render_tests is awesome.
