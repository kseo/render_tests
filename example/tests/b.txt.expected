cmVuZGVyX3Rlc3RzIGlzIGF3ZXNvbWUuCg==